rules/code-quality/tests/Rector/BooleanNot/SimplifyDeMorganBinaryRector/Fixture/skip_greater_or_equal.php.inc

```diff
@@ -0,0 +1,15 @@
+<?php
+
```

Comment:
Here should be namespace of the fixture as well.

Reply:
fixed.